packages/widget/src/hooks/useCreateCosmosWallets.tsx

```diff
@@ -29,12 +29,14 @@ import { useCallback } from "react";
 import { skipAssetsAtom, skipChainsAtom } from "@/state/skipClient";
 import { sourceAssetAtom } from "@/state/swapPage";
 import { isMobile } from "@/utils/os";
+import { allCallbacksAtom } from "@/state/callbacks";
```

Comment:
In terms of naming, what do you think about just `callbacks` or `walletCallbacks` ? (for this atom as well as rest of references)

Reply:
agree on this one